agent/pkg/rules/models.go

```diff
@@ -88,32 +42,46 @@ func ValidateService(serviceFromRule string, service string) bool {
 }
 
 func MatchRequestPolicy(harEntry har.Entry, service string) (int, []RulesMatched) {
-	enforcePolicy, _ := DecodeEnforcePolicy()
+	enforcePolicy, _ := shared.DecodeEnforcePolicy("/app/enforce-policy/enforce-policy.yaml")
```

Comment:
It's great that you moved DecodeEnforcePolicy to shared 👍 
Can you also add `"/app/enforce-policy/enforce-policy.yaml"`?

Reply:
put this into shared? I think not, because I'm using this function on CLI side

Reply:
The file path is defined in both agent (`/app/enforce-policy/enforce-policy.yaml`) and in CLI (`/app/enforce-policy`, `enforce-policy.yaml`).
If in the future someone wants to change the file name, then they will need to do it in two places. If it was defined with consts in shared, then they would only need to change it in one place.